api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto

```diff
@@ -60,9 +67,7 @@ message LeastRequest {
   // Configuration for local zone aware load balancing or locality weighted load balancing.
   common.v3.LocalityLbConfig locality_lb_config = 4;
 
-  // [#not-implemented-hide:]
-  // Configuration for performing full scan on the list of hosts.
-  // If this configuration is set, when selecting the host a full scan on the list hosts will be
-  // used to select the one with least requests instead of using random choices.
-  google.protobuf.BoolValue enable_full_scan = 5;
+  // Method for selecting the host set from which to return the host with the fewest active requests.
```

Comment:
```suggestion
  // Method for selecting the host set from which to return the host with the fewest active requests.
  //
```

assuming you want a newline here (i think so)

Reply:
Added the newline